Add docs for Visual Studio Extension (#3672)
conan-io/docs: integrations/visual_studio.rst

:ref:`reference <conan_tools_microsoft>` section.

Conan extension for Visual Studio
---------------------------------

There's an extension `available in the VisualStudio Marketplace
<https://marketplace.visualstudio.com/items?itemName=conan-io.conan-vs-extension>`_ that's

> compatible with Visual Studio versions higher than *2022*. With this extension, you can

Reviewer: ```suggestion
compatible begining from Visual Studio  version *2022*. With this extension, you can
```
